tokio-threadpool/src/blocking.rs

```diff
@@ -29,8 +29,8 @@ pub struct BlockingError {
 ///
 /// # Return
 ///
-/// When the blocking closure is executed, `Ok(T)` is returned, where `T` is the
-/// closure's return value.
+/// When the blocking closure is executed, `Ok(Ready(T))` is returned, where
```

Comment:
This change looks wrong from a cursory glance.

Reply:
The function returns `Poll<T, BlockingError>`, which is equivalent to `Result<Async<T>, BlockingError>`.

On success, we can return either `Ok(Async::Ready(T))` or `Ok(Async::NotReady)`. A sentence several lines down mentions the latter case.

Reply:
Ah, right, sorry.  Had a brainfart and was translating `Poll<T, _>` as `Result<T, _>` in my head. :)